src/main/java/uk/nhs/hee/tis/trainee/sync/config/AmazonSQSConfig.java

```diff
@@ -0,0 +1,16 @@
+package uk.nhs.hee.tis.trainee.sync.config;
+
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AmazonSQSConfig {
```

Comment:
This will get flagged in the style checker for incorrect casing, should be `AmazonSqsConfig`. Same issue with the method name.

See https://google.github.io/styleguide/javaguide.html#s5.3-camel-case